embed/src/main/java/orca/embed/cloudembed/controller/InterCloudHandler.java

```diff
@@ -139,7 +139,11 @@ else if(domain.equals(RequestReservation.Interdomain_Domain)){  //call @ InterDo
 			s_rr.remove(RequestReservation.MultiPoint_Domain);
             request.getElements().removeAll(mp_elements);
             for(NetworkElement ne:this.mpRequest.getElements()){
-            	//ne.setInDomain(RequestReservation.Interdomain_Domain);
+				// #162 make sure the element has not been Closed before trying to add it
```

Comment:
@hinchliff - indentation feels odd here? am I missing something?

Reply:
Some of the code is tabs, some of the code is spaces...  (If you view this in an editor with a tab size of 4, it _looks_ right...)

I think the team is in agreement that spaces should be used.  But a lot of the old code still has tabs in it, and we haven't (yet) applied a consistent code style to the project.